lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -521,7 +521,7 @@ def add_xid(xml, creditcard, three_d_secure)
       end
 
       def add_cavv(xml, creditcard, three_d_secure)
-        return unless three_d_secure && creditcard.brand == 'visa'
+        return unless three_d_secure && ['visa', 'discover'].include?(creditcard.brand)
```

Comment:
This matches the existing code at line 497.
```suggestion
        return unless three_d_secure && %w(visa discover).include?(creditcard.brand)
```